inc/uf2.h.orig

```diff
@@ -0,0 +1,278 @@
+#ifndef UF2_H
```

Comment:
I imagine this one was not meant to be checked in?

Reply:
No :). I just removed it. I was just actually going to push a change to this PR in the next five minutes or so.